__rebuildDatabase.py

```diff
@@ -0,0 +1,50 @@
+import mysql.connector
+import os
+
+try:
+    # Connect to the MySQL database
+    host = os.getenv('DB_HOST', 'localhost')
+    user = os.getenv('DB_USER', 'root')
+    password = os.getenv('DB_PASSWORD', 'root')
+    port = os.getenv('DB_PORT', 8889)
+
```

Comment:
# I experienced an error when I tried to follow the set up dev environment process. after some troubleshooting with chat gpt, it seems it might be because the name of the database is not specified in this file. doing it here would look something like:

database_name = "WheelDeal"